examples/puppeteer/oauth2-test.js

```diff
@@ -0,0 +1,117 @@
+// Copyright 2018, Google, Inc.
```

Comment:
Google LLC. The copyright template was updated with Google LLC.

Reply:
fixed!